materials/version-control-R.md

```diff
@@ -274,16 +252,8 @@ add_size_classes <- function(df) {
 > Have students email a link to their repo to their instructor once they have
 > finished Pushing Changes
 >
-> The instructor should then commit the following code to their repo
->
-> get_size_class <- function(ear_length){
->   # Calculate the size class for one or more ear lengths
->   ear_lengths <- ifelse(ear_length > 10, "large", "small")
->   return(ear_lengths)
-> }
->
-> With the commit message:
-> Add function for determining ear length class
+> The instructor should then commit the *Add output table* code to their repo
```

Comment:
I'm a little lost here. Are you saying to add the same code that we pushed above? If so, then I think we should anticipate that most of the students will have already done this while following along and so have them get something different from what they've already (potentially) added.

I don't think this every ended up in the notes, but ideally we'd have an assistant during the lecture who pushes something that the instructor pulls. Then the students should all pull the same thing. Does that make sense?